xml/Accessibility/CAccPropServicesClass.xml

```diff
@@ -263,7 +269,7 @@
       <MemberSignature Language="F#" Value="abstract member ComposeHwndIdentityString :  * uint32 * uint32 * nativeint *  -&gt; unit&#xA;override this.ComposeHwndIdentityString :  * uint32 * uint32 * nativeint *  -&gt; unit" Usage="cAccPropServicesClass.ComposeHwndIdentityString (hwnd, idObject, idChild, ppIDString, pdwIDStringLen)" />
       <MemberType>Method</MemberType>
       <Implements>
-        <InterfaceMember>M:Accessibility.IAccPropServices.ComposeHwndIdentityString(Accessibility._RemotableHandle@,System.UInt32,System.UInt32,System.IntPtr,System.UInt32@)</InterfaceMember>
+        <InterfaceMember>M:Accessibility.IAccPropServices.ComposeHwndIdentityString(Accessibility._RemotableHandle,System.UInt32,System.UInt32,System.IntPtr,System.UInt32)</InterfaceMember>
```

Comment:
The Implements section disappeared with this change. By design?
https://review.docs.microsoft.com/en-us/dotnet/api/accessibility.caccpropservicesclass.composehwndidentitystring?view=netframework-4.7.2&branch=CITest#Accessibility_CAccPropServicesClass_ComposeHwndIdentityString_Accessibility__RemotableHandle__System_UInt32_System_UInt32_System_IntPtr_System_UInt32__

Reply:
@mairaw to clarify the question ... the `<Implements>` section is still preset in the EcmaXml, and you're asking why it's disappeared on the OPS front end, right? More of a question for @dend :)

Reply:
@mairaw @dend wait wait ... On second review, I noticed a subtle change on this line, where the `ref` parameter had a `@` removed. I'm **guessing** that the front end probably does a docid lookup before displaying this section, and it couldn't find a match. 

I've created a GH issue to address/fix this, and it will be in the next release: https://github.com/mono/api-doc-tools/issues/304